src/Compilers/CSharp/Portable/Binder/Binder_Symbols.cs

```diff
@@ -1524,6 +1524,41 @@ internal static bool TryGetSpecialTypeMember<TSymbol>(CSharpCompilation compilat
             return true;
         }
 
+#nullable enable
+
+        /// <summary>
+        /// This is a layer on top of the Compilation version that generates a diagnostic if the well known
+        /// member isn't found.
+        /// </summary>
+        internal Symbol? GetWellKnownTypeMember(WellKnownMember member, BindingDiagnosticBag diagnostics, SyntaxNode syntax)
```

Comment:
>GetWellKnownTypeMember [](http://example.com/codeflow?start=25&length=22)

Is this different in behavior from the `GetWellKnownTypeMember()` overload at line 1645?